src/evict/evict_lru.c

```diff
@@ -2884,13 +2870,13 @@ __wti_evict_app_assist_worker(WT_SESSION_IMPL *session, bool busy, bool readonly
          * Additionally we don't return rollback which could confuse the caller.
          */
         if (__wt_op_timer_fired(session))
-            break;
+            goto err;
```

Comment:
I'd like how it was back when it was using break instead here. That's my preference, feel free to ignore it.

Reply:
I replaced these breaks with gotos to differentiate them from the breaks in the switch-case below, which actually continue the loop rather than exiting it.